backend/dashboard/views/pages/Billing.vue

```diff
@@ -0,0 +1,19 @@
+<template lang='pug'>
+div
+  h1.c-page-header Billing
+
+  p this is 'billing' page
```

Comment:
Let's include the i18n stack in this project too, so that everything is setup to be translated from the start. We don't want language to be a barrier to using Chelonia. :)

For now OK to just copy the files. In the future we need to separate out i18n stuff into its own library.